### PR TITLE
Update how to remove trusted clusters.

### PR DESCRIPTION
**Pull request by benarent**
Extra docs from https://github.com/gravitational/teleport/issues/2075  on how to disable trusted clusters.

### CODE REVIEW

**Comment by webvictim** [comment]
This is already in the admin guide: https://gravitational.com/teleport/docs/admin-guide/#disabling-trust

Maybe we just update that rather than having a separate section? Or certainly fix that part too.

**Comment by benarent** [comment]
I just went ahead and removed / refactored the admin guide and placed most of the contents on the Trusted Cluster page since we had a lot of duplicate content.

File: docs/4.3/trustedclusters.md [hunk 1]
```diff
@@ -388,16 +484,16 @@ There are three common types of problems Teleport administrators can run into wh
...
+  `tsh clusters` output on "root".
+* **Access problems**: when users from "root" get "access denied" error messages
+  trying to connect to nodes on "leaf".
 
 ### HTTPS configuration
 
 If the `web_proxy_addr` endpoint of the main cluster uses a self-signed or invalid HTTPS certificate,
 you will get an error: "the trusted cluster uses misconfigured HTTP/TLS certificate". For ease of
-testing the teleport daemon of "east" can be started with  `--insecure` CLI flag to accept
+testing the teleport daemon of "leaf" can be started with  `--insecure` CLI flag to accept
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:496]:
```suggestion
testing, the teleport daemon on "leaf" can be started with the `--insecure` CLI flag to accept
```

File: docs/4.3/trustedclusters.md [hunk 2]
```diff
@@ -388,16 +484,16 @@ There are three common types of problems Teleport administrators can run into wh
...
-* **Access problems**: when users from "main" get "access denied" error messages
-  trying to connect to nodes on "east".
+* **Connectivity problems**: when a trusting cluster "leaf" does not show up in
+  `tsh clusters` output on "root".
+* **Access problems**: when users from "root" get "access denied" error messages
+  trying to connect to nodes on "leaf".
 
 ### HTTPS configuration
 
 If the `web_proxy_addr` endpoint of the main cluster uses a self-signed or invalid HTTPS certificate,
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:494]:
```suggestion
If the `web_proxy_addr` endpoint of the root cluster uses a self-signed or invalid HTTPS certificate,
```

File: docs/4.3/trustedclusters.md [hunk 3]
```diff
@@ -59,13 +60,13 @@ even need to have a direct connection to other clusters' proxy servers. The user
...
+# login using the root "root" cluster credentials:
 $ tsh login --proxy=main.example.com
 
-# SSH into some host inside the "main" cluster:
+# SSH into some host inside the "root" cluster:
 $ tsh ssh host
 
-# SSH into the host located in another cluster called "east"
+# SSH into the host located in another cluster called "leaf"
 # The connection is established through main.example.com:
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:70]:
```suggestion
# The connection is established through root.example.com:
```

File: docs/4.3/trustedclusters.md [hunk 4]
```diff
@@ -156,31 +231,31 @@ more complicated.
...
-  restricted access to "east". We want to deny them access to machines
+* We want administrators from "root" (but not regular users!) to have
+  restricted access to "leaf". We want to deny them access to machines
   with "environment=production" label.
 
-First, we need to create a special role for main users on "east":
+First, we need to create a special role for main users on "leaf":
 
 ```yaml
 # save this into main-user-role.yaml on the east cluster and execute:
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:261]:
```suggestion
# save this into root-user-role.yaml on the leaf cluster and execute:
```

File: docs/4.3/trustedclusters.md [hunk 5]
```diff
@@ -272,10 +346,9 @@ Then following updating the role map, we can re-create the cluster by executing:
...
 ```
 
-
 ## Using Trusted Clusters
 
-Now an admin from the main cluster can now see and access the "east" cluster:
+Now an admin from the main cluster can now see and access the "leaf" cluster:
 
 ```bsh
 # login into the main cluster:
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:354]:
```suggestion
# log into the root cluster:
```

File: docs/4.3/trustedclusters.md [hunk 6]
```diff
@@ -298,30 +371,53 @@ $ tsh ls --cluster=east
...
+
+Remove the relationship from the root cluster: `tctl rm rc/leaf.example.com`.
+
+!!! Note
+
+    The `leaf.example.com` cluster will continue to try and ping the root cluster,
+    but will not be able to connect. To re-establish the trusted cluster relationship,
+    the trusted cluster has to be create again from the leaf cluster.
+
+### Remote Leaf Cluster relationship from the Leaf.
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:417]:
```suggestion
### Remove Leaf Cluster relationship from the leaf
```

File: docs/4.3/trustedclusters.md [hunk 7]
```diff
@@ -388,16 +484,16 @@ There are three common types of problems Teleport administrators can run into wh
 trust between two clusters:
 
 * **HTTPS configuration**: when the main cluster uses a self-signed or invalid HTTPS certificate.
```
Comment (webvictim) [inline docs/4.3/trustedclusters.md:486]:
```suggestion
* **HTTPS configuration**: when the root cluster uses a self-signed or invalid HTTPS certificate.
```

**Review by webvictim — COMMENTED** [review]
Made a lot of main -> root changes. Also "trusting cluster" becomes "leaf cluster" and "trusted cluster" becomes "root cluster".

We still need to go and edit all these "main" references directly in the trusted cluster file as there are a lot of them. I'll wait for you to apply these corrections, then I can go and clean up the rest on the branch if you want.

**Review by webvictim — APPROVED** [review]

**Pull request merged**